build.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const fs = require('fs');
+const http = require('follow-redirects').http;
+const archiver = require('archiver');
+
+var options = {
+    host: 'uplift.shipit.staging.mozilla-releng.net',
+    path: '/coverage/supported_extensions',
+    method: 'GET',
+    headers: {"Content-Type": "text/plain"}
+}
+
+
+var req = http.request(options, (res) => {
```

Comment:
Can you use the `get` method directly, instead of the lower-level `request`?

Also, `https` instead of `http`.